rides-service/src/test/java/com/example/rides_service/unit/service/RideServiceImplTest.java

```diff
@@ -103,30 +110,37 @@ void addRidePassengerNotFound() {
     class EditRideTests {
         @Test
         void editRideOk() {
-            RequestRide requestRide = RideTestEntityUtils.createUpdateRequestRide();
-            Ride updatedRide =  RideTestEntityUtils.createTestUpdatedRide();
-            ResponseRide responseRide = RideTestEntityUtils.createTestUpdatedResponseRide();
-            when(rideRepository.findById(rideId)).thenReturn(Optional.of(testRide));
-            when(rideRepository.save(testRide)).thenReturn(updatedRide);
-            when(rideMapper.rideToResponseRide(updatedRide)).thenReturn(responseRide);
+            RequestRide requestRide = RideTestEntityUtils.createTestRequestRide().driverId(RideTestEntityUtils.DEFAULT_DRIVER_ID).build();
```

Comment:
В случае с билдером, лучше переносить на новые строки, когда устанавливаешь значения в поля
```suggestion
            RequestRide requestRide = RideTestEntityUtils.createTestRequestRide()
            .driverId(RideTestEntityUtils.DEFAULT_DRIVER_ID)
            .build();
```